services/user/Evaluations/ui/src/components/new-eval.tsx

```diff
@@ -0,0 +1,225 @@
+import { useCreateEvaluation } from "@hooks/app/use-create-evaluation";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { useAppForm } from "./app-form";
+import { humanize } from "@lib/humanize";
+import { NumbersField } from "./numbers-field";
+
+dayjs.extend(duration);
+
+const lowestFifth = (minute: number) => minute - (minute % 5);
+
+const now = dayjs();
+
+const registration = now;
+const deliberation = now.add(10, "seconds");
+const submission = deliberation.add(30, "seconds");
+const finishBy = submission.add(10, "seconds");
+
+export const NewEval = ({ onSubmit }: { onSubmit: () => void }) => {
+    const { mutateAsync: createEvaluation } = useCreateEvaluation();
+
+    const form = useAppForm({
+        defaultValues: {
+            registration: registration.toDate(),
+            deliberation: deliberation.toDate(),
+            submission: submission.toDate(),
+            finishBy: finishBy.toDate(),
+            allowableGroupSizes: [1, 2, 3, 4, 5, 6],
+            numOptions: 6,
+        },
+        validators: {
+            onSubmit: (data) => {
+                const now = dayjs();
+                const { deliberation } = data.value;
+
+                if (now.valueOf() > deliberation.valueOf()) {
+                    return "Deliberation phase has already started";
```

Comment:
When trying to create an evaluation, I'm getting this error as I try to change the deliberation phase to start a few minutes later (instead of in 10 seconds from the default).